akka-actor-typed-tests/src/test/scala/docs/akka/typed/IntroSpec.scala

```diff
@@ -190,7 +190,7 @@ object IntroSpec {
             handle ! PostMessage("Hello World!")
             Behaviors.same
           case MessagePosted(screenName, message) =>
-            context.log.info("message has been posted by '{}': {}", screenName, message)
+            context.log.info("message has been posted by '{}': {}", screenName, message: Any)
```

Comment:
That is annoying. We should talk to the Scala team about it.

Reply:
fixed (or at least improved) in #27605

I'm not sure this is a Scala problem, the Java API is looking for trouble by defining both:
```
public void info(String format, Object arg1, Object arg2);
public void info(String format, Object... arguments);
```

I guess it would have been better it was
```
public void info(String format, Object arg1, Object arg2, Object... arguments);
```

Reply:
Very weird  that they have conflicting signatures like that.